superset/views/core.py

```diff
@@ -1968,6 +1970,48 @@ def sync_druid_source(self):
             return json_error_response(utils.error_msg_from_exception(e))
         return Response(status=201)
 
+    @has_access
+    @expose("/sqllab_table_viz/", methods=["POST"])
```

Comment:
What happens if you try to create a table that already exists with a different schema?

Reply:
I think this endpoint should probably have a name that reflects what it does, rather than being sqllab-specific.

Reply:
RE: What happens if you try to create a table that already exists with a different schema?
- it return the found value. 

Good suggestion with the name, renamed the endpoint.